Sources/HttpPipeline/SignedCookies.swift

```diff
@@ -0,0 +1,148 @@
+import CommonCrypto
+import Cryptor
+import Foundation
+
+extension ResponseHeader {
+  /// Computes a signed cookie.
+  ///
+  /// - Parameters:
+  ///   - key: The name of the cookie.
+  ///   - value: The data of the cookie to sign.
+  ///   - options: (Optional) Extra options to attach to the cookie.
+  ///   - secret: The secret to sign the cookie with. This value must also be provided to verify a signature.
+  ///   - encrypt: (Optional) Further encrypts the signed cookie using the secret provided.
+  /// - Returns: A `Set-Cookie` header containing the signed cookie.
+  public static func setSignedCookie(
```

Comment:
these are static funcs on `ResponseHeader` so you can use it like you would `writeHeader(.setCookie(...))`